CONTRIBUTING.md

```diff
@@ -18,27 +18,24 @@ In case of questions about the contribution process or for discussion of specifi
 
 For a Pull Request to be considered at all it has to meet these requirements:
 
-1. Pull Request branch should be given a unique [descriptive name](#pull-request-branch-naming) that explains its intent.
+1. Pull Request branch should be given a unique descriptive name that explains its intent.
 2. Code in the branch should live up to the current code standard:
    - Not violate [DRY](http://programmer.97things.oreilly.com/wiki/index.php/Don%27t_Repeat_Yourself).
    - [Boy Scout Rule](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule) needs to have been applied.
 3. Regardless if the code introduces new features or fixes bugs or regressions, it must have comprehensive tests.
 4. The code must be well documented (see the [Documentation](#documentation) section below).
 5. The commit messages must properly describe the changes, see [further below](#creating-commits-and-writing-commit-messages).
 6. Do not use ``@author`` tags since it does not encourage [Collective Code Ownership](http://www.extremeprogramming.org/rules/collective.html). Contributors get the credit they deserve in the release notes.
+7. Pull Request comment in GitHub must have a reference to the issue that the Pull Request is addressing, e.g. "Fixes #1337".
```

Comment:
I don't see this as a hard requirment, especially not for those smaller community projects. In the end I want a list of what is included in a release. That list can be issues or pull requests marked with the milestone of the release.

Small fixes can be just a pull request that is marked with the milestone.

Essential features or important bug fixes should have an issue and then only the issue is marked with the milestone, not the pull request.